lib/team_collection.rb

```diff
@@ -0,0 +1,17 @@
+require './lib/team'
+require 'CSV'
+
+class TeamCollection
+  attr_reader :teams
+
+  def initialize(team_file_path)
+    @teams = create_teams(team_file_path)
+  end
+
+  def create_teams(team_file_path)
```

Comment:
This seems like a good place for inheritance or a module in the future.  But works for now!

Reply:
I'll second that motion!